kube/app/ingress.yml

```diff
@@ -13,7 +13,7 @@ metadata:
   {{ end }}
   annotations:
     ingress.kubernetes.io/backend-protocol: "HTTPS"
-    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    ingress.kubernetes.io/force-ssl-redirect: "true"
     kubernetes.io/ingress.class: "nginx-external"
```

Comment:
I think this can be removed. And the following added in the ingress file
`ingressClassName: nginx-external`. It goes under spec: see what I've done here
https://github.com/UKHomeOffice/asylum-contact-questionnaire/blob/master/kube/app/ingress-external.yml

Reply:
<img width="982" alt="Screenshot 2023-05-11 at 16 38 19" src="https://github.com/UKHomeOffice/home-office-forms-demo/assets/122998307/98ae9e66-c219-4c31-aa9b-818935578fb3">

like this?

Reply:
No I meant you need to remove `kubernetes.io/ingress.class: "nginx-external"` too

Reply:
yeh i figured, i removed it in my latest commit :)